### PR TITLE
ACS event updates

### LINKED ISSUE #19603: AcsChatParticipantAddedToThreadEvent/AcsChatParticipantRemovedFromThreadEvent system event mapping is broken resulting in Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events not being usable in the SDK
**Describe the bug**
AcsChatParticipantAddedToThreadEvent/AcsChatParticipantRemovedFromThreadEvent system event mapping is broken resulting in Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events not being usable in the SDK

**Expected behavior**
Successfully able retrieve event payload Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved 

**Actual behavior (include Exception or Stack Trace)**
What is the actual behavior?
SDK rejects the payload due to missing system event mapping

**To Reproduce**
Steps to reproduce the behavior (include a code snippet, screenshot, or any additional information that might help us reproduce the issue)

1.Attempt to parse Microsoft.Communication.ChatThreadParticipantAdded/Microsoft.Communication.ChatThreadParticipantRemoved events using the SDK
2.

**Environment:**
 - Name and version of the Library package used: [e.g. Azure.Storage.Blobs 12.2.0] 
 - Hosting platform or OS and .NET runtime version (`dotnet --info` output for .NET Core projects): [e.g. Azure AppService or Windows 10 .NET Framework 4.8]
 - IDE and version : [e.g. Visual Studio 16.3]

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Fixes https://github.com/Azure/azure-sdk-for-net/issues/19603

### CODE REVIEW

File: sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static class SystemEventNames
         /// The value of the Event Type stored in <see cref="EventGridEvent.EventType"/> and <see cref="CloudEvent.Type"/> for the
         /// <see cref="AcsChatParticipantAddedToThreadEventData"/> system event.
         /// </summary>
-        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatParticipantAddedToThread";
+        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatThreadParticipantAdded";
```
Comment (ellismg) [inline sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs:35]: Feels like we need to add something for `Microsoft.Communication.RecordingFileStatusUpdated` to this file?

**Review by ellismg — CHANGES_REQUESTED** [review]
I think you might need to add something to SystemEventNames.cs for the new event?

File: sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs (hunk 1 above)
Comment (JoshLove-msft) [reply sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs:35]: Yep, looks like the test actually works! https://dev.azure.com/azure-sdk/public/_build/results?buildId=796869&view=ms.vss-test-web.build-test-results-tab&runId=17558288&resultId=100009&paneView=debug

**Review by ellismg — APPROVED** [review]
LGTM - It might be nice to put a little more detail in the commit message about what the problem was and how we decided to fix it (not changing the names of the properties but changing their values, because we didn't want to introduce a break), which may be helpful for folks down the line who notice the inconsistency, run and run `git blame` to try to figure out why things are the way they are.

Very cool that you had that test that failed when we forgot to add the event to the mapping table, great job on that.

**Comment by JoshLove-msft** [comment]
> LGTM - It might be nice to put a little more detail in the commit message about what the problem was and how we decided to fix it (not changing the names of the properties but changing their values, because we didn't want to introduce a break), which may be helpful for folks down the line who notice the inconsistency, run and run `git blame` to try to figure out why things are the way they are.
> 
> Very cool that you had that test that failed when we forgot to add the event to the mapping table, great job on that.

Sure updated - for things like this I usually end up needing to look at the PR that made the change, but any additional breadcrumbs we can add should help as well.

**Pull request merged**